content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,33 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following PersistentVolume types are supported through the `CSIMigration` feature gate and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  &mdash; but for Kubernetes version 1.27 and onwards, this volume type is **only** available via a separate CSI driver 
+  as Kubernetes' core no longer provides built-in support. The Kubernetes project provides an equivalent
+  CSI driver, [AWS EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver).
 * [`azureDisk`](/docs/concepts/storage/volumes/#azuredisk) - Azure Disk
-  (**not available** in v1.27)
+  &mdash; but for Kubernetes version 1.27 and onwards, this volume type is **only** available via a separate CSI driver
```

Comment:
AzureDisk CSI migration is [GA since 1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#feature-17)

I.e. 1.24 already required the CSI driver  to be installed. In 1.27 we only removed the in-tree code for AWS EBS, but it has been (mostly) unused for 2 releases already.

Reply:
The documentation is capturing the version since the use of the CSI driver was mandatory. So 1.27 is correct, right?

Reply:
Updated the version in the latest commit.